### ISSUE TITLE
Text rows not UTF-8 decoded when read from server

### ISSUE THREAD

**Issue by enginoid** [finagle/roc#70]
Rows that return unicode strings come back in a different formatting than expected. Where the string in the database is "Empeñada", I get back "Empeￃﾱada".

Here's an example method:

```scala
def getFood(): Future[String] = {
  val req = new Request(s"SELECT 'Empeñada' as course".stripMargin)
  pgClient.query(req).map { result =>
    result.map(_.get('course).as[String]).head
  }
}
```

Here's how I called it:

```
scala> getFood().get
res5: String = Empeￃﾱada
```

Which we can verify is not just my terminal:

```
scala> food.foreach(char => println("%02X" format char.hashCode))
45
6D
70
65
FFC3
FFB1
61
64
61
```

@penland365 believes the issue is in [Result.rows](https://github.com/finagle/roc/blob/master/core/src/main/scala/roc/postgresql/results.scala#L30-L31
) where this:

```scala
val strValue = bytes.map(_.toChar).mkString
Text(column.name, column.columnType, strValue)
```

...should be something like this:

```scala
val strValue = new String(bytes, StandardCharsets.UTF_8)
Text(column.name, column.columnType, strValue)
```

**Comment by penland365** [comment]
Closed by #71 . Thanks again @enginoid !

**Comment by enginoid** [comment]
Thanks for fixing this so quickly!
